### PR TITLE
Move Bugzilla config data into a separate JSON file.

### PR DESCRIPTION
**Pull request by markrcote**
This will make it easy for other apps to pull down the dashboard's current config, and it keeps config updates separate from code changes.

Specifically, I'm looking to use this for an IRC bot, at @dougt's request, and I don't want to maintain two copies of the current dashboard config.

### CODE REVIEW

File: js/bzconfig.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "BUGZILLA_URL": "https://bugzilla.mozilla.org/buglist.cgi",
+  "BUGZILLA_REST_URL": "https://bugzilla.mozilla.org/rest/bug",
+  "versions": {
+    "release": {
+      "version": 45,
+      "title": "Firefox",
+      "img": "images/firefox.png"
+    },
+	"beta": {
```
Comment (mnoorenberghe) [inline js/bzconfig.json]: Nit: The indentation of this file seems off

Comment (markrcote) [inline js/bzconfig.json]: Hm yeah I have a mix of spaces and tabs. I'm used to spaces, but the JS here is tab-indented, so apparently I mixed up the two in the new file. To match the original style, I'll tab-indent it.

**Comment by markrcote** [comment]
Not terribly clear here, but I pushed an update switching all indentation in bzconfig.js to tabs, just to match the rest of the app. I've got another PR coming (based on this one) to switch everything to spaces. ;)

**Comment by markrcote** [comment]
Easier to just tack it on here.

**Comment by dklawren** [comment]
Manually merged pull request to resolve conflict.

To https://github.com/mozilla/releasehealth.git
   445a742..8c9a555  master -> master

**Pull request closed without merging**